### PR TITLE
Change theme to SymbiFlow

### PR DESCRIPTION
**Pull request by mgielda**

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -113,7 +113,7 @@
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
 #
-html_theme = 'sphinx_rtd_theme'
+html_theme = 'sphinx_symbiflow_theme'
```
Comment (mithro) [inline docs/conf.py]: Does this need to be installed somehow?

Comment (mgielda) [reply docs/conf.py]: Yes, it's in the requirements.txt

Comment (mgielda) [reply docs/conf.py]: Just like sphinx_rtd_theme was. It's a 1:1 replacement

Comment (mithro) [reply docs/conf.py]: You need to change the requirements.txt file then?

Comment (mgielda) [reply docs/conf.py]: Yup, forgot to commit the requirements file, sorry

**Pull request merged**